openvpn/primitives/generate.go

```diff
@@ -0,0 +1,134 @@
+package primitives
+
+import (
+	"crypto/ecdsa"
+	"crypto/x509"
+	"fmt"
+	log "github.com/cihub/seelog"
+	"os"
+	"path/filepath"
+)
+
+// SecurityPrimitives describes security primitives
+type SecurityPrimitives struct {
```

Comment:
Is this class "security primitives" itself, or does it generate "security primitives", i.e. "SecurityPrimitivesGenerator"?